py/desitarget/lyazcat.py

```diff
@@ -0,0 +1,441 @@
+"""
+desitarget.lyazcat
+==============
+
+Post-redrock ML processing for LyA Quasar object identification.
+"""
+
+import os
+import numpy as np
+import sys
+from astropy.io import fits
+import fitsio
+import time
+
+from desitarget.geomask import match, match_to
+from desitarget.internal import sharedmem
+from desitarget import io
+from desispec.io import read_spectra
+
+from quasarnp.io import load_model
+from quasarnp.io import load_desi_coadd
+from quasarnp.utils import process_preds
+from squeze.model import Model
+from squeze.common_functions import save_json, load_json
+from squeze.candidates import Candidates
+from squeze.desi_spectrum import DesiSpectrum
+from squeze.spectra import Spectra
+
+# ADM set up the DESI default logger.
+from desiutil.log import get_logger
+log = get_logger()
+
+zcatdatamodel_names = np.array(['TARGETID','Z','ZWARN','DELTACHI2','Z_COMB',
+                                'Z_COMB_PROB','Z_QN','Z_QN_CONF','IS_QSO_QN',
+                                'Z_SQ','Z_SQ_CONF','Z_ABS','Z_ABS_CONF'])
+"""
+zcatdatamodel_formats = np.array(['>int64','>float64','>int64','>float64',
+                                  '>float64','>float64','>float64','>float64',
+                                  '>int16','>float64','>float64','>float64',
+                                  '>float64'])
+"""
+zcatdatamodel_formats = np.array(['>i8','>f8','>i8','>f8',
```

Comment:
Code style. Need a white-space after every comma in a list.

Reply:
Fixed for the next push.